var/spack/repos/builtin/packages/dataspaces/package.py

```diff
@@ -3,75 +3,33 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-import six
-
 from spack import *
 
 
-def is_string(x):
-    """validate a string"""
-    try:
-        return isinstance(x, six.string_types)
-    except ValueError:
-        return False
-
-
-class Dataspaces(AutotoolsPackage):
-    """an extreme scale data management framework."""
-
-    homepage = "http://www.dataspaces.org"
-    url      = "https://dataspaces.rdi2.rutgers.edu/downloads/dataspaces-1.6.2.tar.gz"
-
-    version('1.8.0', sha256='7f204bb3c03c2990f5a2d76a29185466b584793c63ada03e5e694627e6060605')
-    version('1.6.2', sha256='3c43d551c1e8198a4ab269c83928e1dc6f8054e6d41ceaee45155d91a48cf9bf')
-
-    variant('dimes',
-            default=False,
-            description='enabled DIMES transport mode')
-    variant('cray-drc',
-            default=False,
-            description='using Cray Dynamic Credentials library')
-    variant('gni-cookie',
-            default='0x5420000',
-            description='Cray UGNI communication token',
-            values=is_string)
-    variant('ptag',
-            default='250',
-            description='Cray UGNI protection tag',
-            values=is_string)
-    variant('mpi',
-            default=True,
-            description='Use MPI for collective communication')
+class Dataspaces(CMakePackage):
+    """DataSpaces v2 based on margo, also called dspaces"""
 
-    depends_on('m4', type='build')
-    depends_on('automake', type='build')
-    depends_on('autoconf', type='build')
-    depends_on('libtool', type='build')
-    depends_on('mpi', when='+mpi')
+    homepage = "wwww.dataspaces.org"
+    url = ""
+    git = "https://github.com/rdi2dspaces/dspaces.git"
 
-    def autoreconf(self, spec, prefix):
-        bash = which('bash')
-        bash('./autogen.sh')
+    maintainers = ['pradsubedi', 'pdavis']
 
-    def setup_build_environment(self, env):
-        if self.spec.satisfies('+mpi'):
-            env.set('CC', self.spec['mpi'].mpicc)
-            env.set('FC', self.spec['mpi'].mpifc)
+    version('master', branch='master', submodules=True)
 
-        env.set('CFLAGS', self.compiler.cc_pic_flag)
+    variant('pybind', default=True, description='build Python bindings')
```

Comment:
By default, we keep `python` variants off in Spack. This avoids pulling in large dependency trees for pure C/C++/Fortran projects. The naming convention we usually use for variants is `python`:
```suggestion
    variant('python', default=False, description='build Python bindings')
```